### PR TITLE
OIIO: bump, cleanup

### PR DESCRIPTION
**Pull request by extrowerk**

### CODE REVIEW

File: media-libs/openimageio/patches/openimageio-1.8.10.patchset [hunk 1]
```diff
@@ -83,5 +83,37 @@ index dacdc2e..4918389 100644
...
+diff --git a/src/libutil/strutil.cpp b/src/libutil/strutil.cpp
+index c26e210..c9dfbf9 100644
+--- a/src/libutil/strutil.cpp
++++ b/src/libutil/strutil.cpp
+@@ -1140,7 +1140,7 @@ Strutil::strtof (const char *nptr, char **endptr)
+     std::locale native; // default ctr gets current global locale
+     char nativepoint = std::use_facet<std::numpunct<char>>(native).decimal_point();
+     // If the native locale uses decimal, just directly use strtof.
+-    if (nativepoint = '.')
++    if (nativepoint == '.')
```
Comment (alaviss) [inline media-libs/openimageio/patches/openimageio-1.8.10.patchset]: This is an upstream bug that will effect everyone, could you please open an PR there with this patch?

Comment (extrowerk) [reply media-libs/openimageio/patches/openimageio-1.8.10.patchset]: https://github.com/OpenImageIO/oiio/pull/1918

**Comment by waddlesplash** [comment]
PR now has conflicts. @extrowerk, can you take a look?

**Comment by extrowerk** [comment]
Do not merge, as it depend on hdf5, wich depends on fortran, wich is broken.

https://github.com/haikuports/haikuports/issues/2651

**Comment by extrowerk** [comment]
gfortran is solved in the meantime, let's check it again...

**Comment by extrowerk** [comment]
Plenty test fails, can somebody check it please?

**Comment by extrowerk** [comment]
Still many test fails.

**Comment by Begasus** [comment]
Superseded by https://github.com/haikuports/haikuports/commit/58ee5066eb98f609089ec8536bad8838e3a494c5

**Pull request closed without merging**